tests/api.c

```diff
@@ -47010,73 +47010,157 @@ static void test_wolfSSL_EVP_PKEY_encrypt(void)
     printf(resultFmt, passed);
 #endif
 }
-static void test_wolfSSL_EVP_PKEY_sign(void)
+static void test_wolfSSL_EVP_PKEY_sign_verify(void)
 {
+#if defined(OPENSSL_EXTRA) || defined(OPENSSL_EXTRA_X509_SMALL)
+    int encs[3] = {0};
 #if defined(OPENSSL_EXTRA) && !defined(NO_RSA) && defined(WOLFSSL_KEY_GEN) && \
     !defined(HAVE_FAST_RSA) && !defined(HAVE_SELFTEST)
 #if !defined(HAVE_FIPS) || (defined(HAVE_FIPS_VERSION) && (HAVE_FIPS_VERSION>2))
     WOLFSSL_RSA* rsa = NULL;
+    encs[0] = EVP_PKEY_RSA;
```

Comment:
All variable declarations need to be at top. Please move the `encs[0] = EVP_PKEY_RSA;` type lines below all the variables.

```
tests/api.c:47025:5: error: ISO C90 forbids mixed declarations and code [-Werror=declaration-after-statement]
47025 |     WOLFSSL_DSA* dsa = NULL;
      |     ^~~~~~~~~~~
tests/api.c:47030:5: error: ISO C90 forbids mixed declarations and code [-Werror=declaration-after-statement]
47030 |     WOLFSSL_EC_KEY* ecKey = NULL;
      |     ^~~~~~~~~~~~~~
tests/api.c:47034:5: error: ISO C90 forbids mixed declarations and code [-Werror=declaration-after-statement]
47034 |     WOLFSSL_EVP_PKEY* pkey = NULL;
      |     ^~~~~~~~~~~~~~~~
```

Reply:
Thanks! Fixed.

@julek-wolfssl I include your ```siglen = keySz``` code into my code to pass the local test, sorry. So I added your name to the commit log.